assets/dev/scss/editor/_flex-icons.scss

```diff
@@ -10,7 +10,10 @@
 	--is-rtl: 0;
 
 	// Rotation direction - Will be `1` for LTR, `-1` for RTL.
-	--rotation-direction: calc( var( --is-ltr ) - var( --is-rtl ) );
+  &:is(.eicon-justify-start-h,.eicon-justify-start-h) {
```

Comment:
There is a small mistake,
It should be:
`&:is(.eicon-justify-start-h,.eicon-justify-end-h)`